README.md

```diff
@@ -20,48 +21,59 @@ function sum(a, b) {
   return a + b;
 } `); // define a function
 
-console.log(await vm.call("sum", 2, 3)) // 5
+console.log(await vm.call("sum", 2, 3)); // 5
 
 vm.close(); // terminate the worker
 ```
 
-Permissions
------------
+## Permissions
 
-* `--unstable` - permission options in worker is an unstable feature.
-* `--allow-read=path/to/worker.ts` - to launch the worker.
+### Default
 
-The code may terminate the worker
----------------------------------
+- `--unstable` - permission options in worker is an unstable feature.
+- `--allow-read=path/to/worker.ts` - to launch the worker.
 
-Although Deno is able to block disk/network access, we can't prevent the code from calling `self.close()`. If this happens, the worker will be terminated. All calls to `vm.run` will timeout.
+### Custom
 
-XXX is not cloneable
---------------------
+- `--allow-[read,write,net,env,sys,run,ffi,import]=...` - user can specify different permissions via `permissionOptions` to adapt to their use cases.
```

Comment:
Move this to FAQ and explain why the `permissions` option should be included in CLI permission settings.